crates/ide/src/highlight_related.rs

```diff
@@ -706,6 +709,60 @@ impl<'a> WalkExpandedExprCtx<'a> {
     }
 }
 
+pub(crate) fn highlight_unsafe_points(
+    sema: &Semantics<'_, RootDatabase>,
+    token: SyntaxToken,
+) -> FxHashMap<EditionedFileId, Vec<HighlightedRange>> {
+    fn hl(
+        sema: &Semantics<'_, RootDatabase>,
+        unsafe_token: Option<SyntaxToken>,
```

Comment:
```suggestion
        unsafe_token: SyntaxToken,
```
we already have the token here